server/sonar-web/src/main/js/api/quality-gates.ts

```diff
@@ -83,46 +91,63 @@ export function copyQualityGate(data: {
   return postJSON('/api/qualitygates/copy', data).catch(throwGlobalError);
 }
 
-export function setQualityGateAsDefault(id: number): Promise<void | Response> {
-  return post('/api/qualitygates/set_as_default', { id }).catch(throwGlobalError);
+export function setQualityGateAsDefault(data: {
+  id: number;
+  organization?: string;
+}): Promise<void | Response> {
+  return post('/api/qualitygates/set_as_default', data).catch(throwGlobalError);
 }
 
-export function createCondition(gateId: number, condition: RequestData): Promise<any> {
-  return postJSON('/api/qualitygates/create_condition', { ...condition, gateId });
+export function createCondition(
+  data: {
+    gateId: number;
+    organization?: string;
+  } & Partial<Condition>
+): Promise<Condition> {
+  return postJSON('/api/qualitygates/create_condition', data);
 }
 
-export function updateCondition(condition: RequestData): Promise<any> {
-  return postJSON('/api/qualitygates/update_condition', condition);
+export function updateCondition(data: { organization?: string } & Condition): Promise<Condition> {
```

Comment:
`Condition` has almost all fields options. Could be nice to type check that all fields provided here.